ooniprobe/AppDelegate.mm

```diff
@@ -24,8 +24,9 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     [[UIBarButtonItem appearance] setTitleTextAttributes:[NSDictionary dictionaryWithObjectsAndKeys: [UIFont fontWithName:@"FiraSans-Regular" size:16],NSFontAttributeName, nil] forState:UIControlStateNormal];
     [[UINavigationBar appearance] setBarTintColor:[UIColor colorWithRGBHexString:color_blue5 alpha:1.0f]];
     [[UINavigationBar appearance] setTranslucent:FALSE];
-    
-    #ifdef RELEASE
+    [[UINavigationBar appearance] setTintColor:[UIColor colorWithRGBHexString:color_white alpha:1.0f]];
+
+  #ifdef RELEASE
```

Comment:
Please, keep the same indent as before.

Reply:
(I mean, I don't like this way of indenting `#ifdef`s, but at least keep it consistent please!)